.github/workflows/lint_website_doc.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint Website Docs
+
+on:
+  pull_request:
+    # paths:
+    #   - "website/**"
+
+jobs:
+  lint-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+      - name: Vale
+        uses: errata-ai/vale-action@v2
+        continue-on-error: true
+        with:
+          files: '["website", "README.md"]'
```

Comment:
We might want to add the plugins doc files, for example https://github.com/cloudquery/cloudquery/tree/89c9b9ae81be9f1c1aabde52fbb1371bdcfd1ff9/plugins/source/aws/docs